package.json

```diff
@@ -34,6 +34,7 @@
     "js-yaml": "^4.1.0",
     "json-loader": "^0.5.7",
     "prettier": "^2.8.0",
+    "rdme": "^8.6.4",
```

Comment:
Do we really need this dependency when we are using their action?

Reply:
Good point, we don't, removed :)